akka-docs-new/src/main/paradox/guide/quickstart.md

```diff
@@ -0,0 +1,113 @@
+* **Use the latest Akka version for new projects (unless you have additional constraints)!**  
+* **Don’t mix Akka versions! You are free to use any Akka version in your project, but you must use 
+    that version for all Akka core projects** In this sample it means that `akka-actor` and `akka-testkit` should 
+    always be of the same version.
+
+### sbt
+
+If you plan to use sbt, your first step is to set up the directory structure for the project. sbt follows the 
+directory layout standard of Maven. Usually, you want to start with the following directories and files:
+
+* `/src`
+  * `/main` (this is where main, production classes live)
+    * `/scala` (this is where Scala classes live)
+    * `/java` (this is where Java classes live if you plan to use Java as well)
+    * `/resources` (this is where non-source files live which are required on the classpath. 
+    Typical example is application.conf which contains the configuration for your application. 
+    We will cover this in detail in CONFIG-SECTION)
+  * `/test`
+    * `/scala` (this is where Scala test and test helper classes live)
+    * `/java` (this is where Java test and test helper classes live if you plan to use Java as well)
+    * `/resources` (this is where non-source files live which are required on the classpath to run tests. 
+    Typically this contains an application.conf that overrides the default configuration for tests. We will 
+    cover this in detail in CONFIG-SECTION)
+  * `project`
+    * `build.properties` ()
+* `build.sbt`
+
+For example, if you have a Scala class `TestClass` in package `com.example.foo` then should go in 
+`/src/main/scala/com/example/foo/TestClass.scala`.
+
+The file `build.sbt` contains the necessary information for sbt about your project metadata and dependencies. 
+For our sample project, this file should contain the following:
+
+```scala
+// build.sbt
+
+name := "intro-akka"
+organization := "intro-akka.organization"
+version := "0.1-SNAPSHOT"
+
+scalaVersion := "2.11.8"
+val AkkaVersion = "2.4.12"
+
+libraryDependencies += "com.typesafe.akka" %% "akka-actor" % AkkaVersion
+libraryDependencies += "com.typesafe.akka" %% "akka-testkit" % AkkaVersion % "test"
+```
+
+This simple file sets up first the project metadata (_name_ and _organization_; we just picked a sample one here). 
+Then we set up the version of the scala compiler we use, then set a variable with the Akka version we intend to
+use (always try to use the latest). 
+
+As the last step, we add two dependencies, `akka-actor` and `akka-testkit`. Note that we used the `AkkaVersion` 
+variable for both dependencies, ensuring that versions are not accidentally mixed and kept in sync when upgrading.
+
+Finally, check that everything works by running `sbt update` from the base directory of your project 
+(where the `build.sbt` file is).
+
+## Setting up your IDE
+
+If you go with IDEA, you usually have flexible means to import project either manually created by one of the 
```

Comment:
Perhaps qualify it with Intellij IDEA or just call it Intellij (which is what most people call it).